### DISCUSSION TITLE
Welcome to Build Discussions!

### DISCUSSION THREAD

**Discussion by lucasgonze** [merico-ai/build#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nikitakoselev** [comment]
Hi. 

My name is Nikita, I am interested in helping your project to grow.

I have failed to fully install the build locally, as this step doesn't work: https://github.com/merico-dev/build#installing-build-backend
I am getting "link not found" in IntelliJ:
![image](https://user-images.githubusercontent.com/6383738/168482993-61c4e076-ec0f-4b47-b7ee-09d91e409575.png)
![image](https://user-images.githubusercontent.com/6383738/168483007-34701694-323b-423d-b823-abfe88f423b1.png)
![image](https://user-images.githubusercontent.com/6383738/168483027-5eefee16-fcdf-43ef-bbc5-13ae818aafe8.png)
